usaspending_api/etl/data_diff_tool.py

```diff
@@ -0,0 +1,42 @@
+from gresearch.spark.diff import *
+
+def data_diff_function(df1, df2):
+    #assumes dataframes as a delta table
+    table1 = spark.read.load(df1)
+    table2 = spark.read.load(df2)
+    return (table1.diff(table2, table1.columns).show)
+
+
+def diff(left: DataFrame, right: DataFrame, unique_key_col="id", compare_cols=[], collect=False):
+    # unique_key_col = "id"
+
+    # If compare_cols is left as an empty list, ALL columns will be compared to each other
+    # (for rows where unique_key_col values match).
+    # Otherwise, specify the columns that determine "sameness" of rows
+    # compare_cols = []
+    # compare_cols = ["value", "v2"]
```

Comment:
Looks like there are some commented out debug statements here. Can you clean lines 11-17 up to only include documentation comments?